homeassistant/components/zwave_mqtt/services.py

```diff
@@ -0,0 +1,173 @@
+"""Methods and classes related to executing Z-Wave commands and publishing these to hass."""
+import logging
+
+from openzwavemqtt.const import CommandClass, ValueType
+import voluptuous as vol
+
+from homeassistant.core import callback
+import homeassistant.helpers.config_validation as cv
+
+from . import const
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class ZWaveServices:
+    """Class that holds our services ( Zwave Commands) that should be published to hass."""
+
+    def __init__(self, hass, manager, data_nodes):
+        """Initialize with both hass and ozwmanager objects."""
+        self._hass = hass
+        self._manager = manager
+        self._data_nodes = data_nodes
+
+    @callback
+    def register(self):
+        """Register all our services."""
+        self._hass.services.async_register(
+            const.DOMAIN,
+            const.SERVICE_ADD_NODE,
+            self.add_node,
+            schema=vol.Schema(
+                {
+                    vol.Optional(const.ATTR_INSTANCE_ID, default=1): vol.Coerce(int),
```

Comment:
These are not used right now right ?

Reply:
Nothing internal to the component uses the add and remove node services, they're there for convenience for adding/removing nodes without having to fire up the qt-openzwave admin tool or sending manual mqtt commands. I'd like to move the add/remote node services to websocket commands once we start building out the new z-wave config panel.